### PR TITLE
console: add more details on console configuration

### PR DESCRIPTION
**Pull request by lucab**
This expands the troubleshooting page, adding more details related
to console settings and showing how to adjust its configuration
permanently.

### CODE REVIEW

File: modules/ROOT/pages/emergency-shell.adoc [hunk 1]
```diff
@@ -1,14 +1,52 @@
-= Access the emergency shell
+= Emergency console access
 
+Sometimes you may want to access the node console to perform troubleshooting steps or emergency maintenance.
+For instance, you may want to access the emergency shell on the console, in order to debug first boot provisioning issues.
```
Comment (jlebon) [inline modules/ROOT/pages/emergency-shell.adoc:4]: I feel like we should mention somewhere around here the semantics that the last console arg wins which is what motivates wanting to modify the kargs when trying to access the emergency console.

**Review by jlebon — APPROVED** [review]
Thanks for adding these details, very helpful!

File: modules/ROOT/pages/emergency-shell.adoc (hunk 1 above)
Comment (lucab) [reply modules/ROOT/pages/emergency-shell.adoc:4]: Ack. I double-checked how this works with https://www.kernel.org/doc/html/v4.14/admin-guide/serial-console.html, then added a second commit with some more details.

**Comment by lucab** [comment]
@jlebon tweaked with a second commit, feel free to merge if it looks better now.

**Review by jlebon — APPROVED** [review]
LGTM!

**Pull request merged**